docs/admin/troubleshooting.md

```diff
@@ -414,6 +414,271 @@ Verify if you're using Infiniband for `fabric_iface`: in the server config. The
 		$ dmg pool destroy --pool=$DAOS_POOL --force
 		Pool-destroy command succeeded
 
+### ipmctl
+
+IPMCTL utility is used for Intel® Optane™ persistent memory for managing, diagnostic and testing purpose.
+[IPMCTL user guide](https://docs.pmem.io/ipmctl-user-guide/) has more details about the utility.
+
+We can use the [diagnostic](https://docs.pmem.io/ipmctl-user-guide/debug/run-diagnostic) and
+[show error log](https://docs.pmem.io/ipmctl-user-guide/debug/show-error-log) functionality to debug the PCMEM related issues.
+
+#### Run a full diagnostic test on clean system
+
+        #ipmctl start -diagnostic
+        --Test = Quick
+           State = Ok
+           Message = The quick health check succeeded.
+           --SubTest = Manageability
+                  State = Ok
+           --SubTest = Boot status
+                  State = Ok
+           --SubTest = Health
+                  State = Ok
+
+        --Test = Config
+           State = Ok
+           Message = The platform configuration check succeeded.
+           --SubTest = PMem module specs
+                  State = Ok
+           --SubTest = Duplicate PMem module
+                  State = Ok
+           --SubTest = System Capability
+                  State = Ok
+           --SubTest = Namespace LSA
+                  State = Ok
+           --SubTest = PCD
+                  State = Ok
+
+        --Test = Security
+           State = Ok
+           Message = The security check succeeded.
+           --SubTest = Encryption status
+                  State = Ok
+           --SubTest = Inconsistency
+                  State = Ok
+
+        --Test = FW
+           State = Ok
+           Message = The firmware consistency and settings check succeeded.
+           --SubTest = FW Consistency
+                  State = Ok
+           --SubTest = Viral Policy
+                  State = Ok
+           --SubTest = Threshold check
+                  State = Ok
+           --SubTest = System Time
+                  State = Ok
```

Comment:
I think that the intent is to provide the most useful command to run a quick diagnostic with expected output and then a more detailed one. It won't be exhaustive, but can be an entry point.